src/libs/actions/Policy.js

```diff
@@ -468,6 +468,72 @@ function update(policyID, values, shouldGrowl = false) {
         });
 }
 
+/**
```

Comment:
I would leave a short Doc here  just for consistency

Reply:
Good point, done!